scripts/columnstore-init

```diff
@@ -2,6 +2,7 @@
 
 # Set Path Variables
 IFLAG=/etc/columnstore/container-initialized
+SHAREDFLAG=/var/lib/columnstore/storagemanager/sm-initialized
```

Comment:
Flag stored in sharedlocation if using shared volumes. Should not break system for non-shared volume usage as each node will only know of it's own /var/lib/columnstore/storagemanager folder if that volume is not shared